src/nv_ingest/extraction_workflows/pdf/pdfium_helper.py

```diff
@@ -176,11 +188,11 @@ def extract_tables_and_charts_using_image_ensemble(
         logger.error(f"Error during table/chart extraction: {str(e)}")
         raise
     finally:
-        if isinstance(paddle_client, grpcclient.InferenceServerClient):
+        if extract_tables and isinstance(paddle_client, grpcclient.InferenceServerClient):
```

Comment:
I don't think you need these extra checks. If extract tables/charts aren't set, the corresponding clients should just be 'None'